node_modules/oae-tenants/tests/test-tenants.js

```diff
@@ -89,7 +87,12 @@ describe('Tenants', function() {
                         // Verify that the global admin tenant is not included
                         assert.ok(!tenants['admin']);
 
-                        callback();
+                        // Check that paging works
+                        RestAPI.Tenants.getTenants(globalAdminRestContext, 0, 1, function(err, tenants) {
```

Comment:
These tests will have to verify that doing `0, 1` gets the first tenant and `1, 1` gets the second tenant